pkg/subctl/benchmark/throughput.go

```diff
@@ -0,0 +1,182 @@
+package benchmark
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("[throughput] Test throughput between two Clusters", func() {
+	f := framework.NewFramework("throughput")
+
+	When("a pod sends throughput test to a pod in a remote cluster", func() {
+		It("should be able to execute throughput test", func() {
+			RunThroughputTest(f)
+		})
+	})
+})
+
+func RunThroughputTest(f *framework.Framework) {
+	clusterAName := framework.TestContext.ClusterIDs[framework.ClusterA]
+	clusterBName := framework.TestContext.ClusterIDs[framework.ClusterB]
+
+	By(fmt.Sprintf("Creating a Nettest Server Pods on %q", clusterBName))
```

Comment:
As part of the output, it would help if we can display info of the node on which the client/server pod is scheduled in the cluster?